src/store/ordersActions.js

```diff
@@ -72,3 +72,27 @@ export function getOrder (id) {
             .catch(error => handleFetchErrors(error, dispatch));
     };
 }
+
+export const signUpForMeal = (orderId, username, what, howMuch) => {
+    const payload = {
+        cost: howMuch,
+        hungryGuy: username,
+        name: what,
+        orderId: orderId
+    };
+
+    return dispatch => {
+        fetch('api/order/meal', {
+            method: 'post',
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(payload)
+        })
+        .then(checkFetchForErrors)
+        .then(response => response.json())
+        .then(() => getOrder(orderId)(dispatch))
```

Comment:
this is heavy, can't we just add a meal to the order on client side only instead of refreshing the whole order? also we would need to reload the order automatically for other users, but that's for the future